<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="5">
    <widget class="QLabel" name="uploadingImageLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="activeButton">
     <property name="text">
      <string>activate</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="loginButton">
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="codeLineEdit">
     <property name="toolTip">
      <string>Paste here the code you get when you login.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Code:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QCheckBox" name="exifCheckBox">
     <property name="text">
      <string>save exif data</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="currentUserLabel">
     <property name="text">
      <string>not logged in!</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
